test/e2e/framework/identityvalidator/identityvalidator_helpers.go

```diff
@@ -56,10 +56,10 @@ func Create(input CreateInput) *corev1.Pod {
 				{
 					Name:  "identity-validator",
 					Image: fmt.Sprintf("%s/identityvalidator:%s", input.Config.Registry, input.Config.IdentityValidatorVersion),
-					Command: []string{
-						"sleep",
-						"3600",
```

Comment:
Same here. Implemented `identityvalidator --sleep` to workaround, which allows the container to sleep forever.